### PR TITLE
Debugger improvements

### PR DESCRIPTION
**Pull request by MRG95**
First pull request and first time using github so rip me if I did it wrong.

I repositioned the debugger's current instruction to be in the middle. It used to be at the top so u always had to scroll to see what came before it. Now it's smack dab in the middle.

Added a breakpoints list so you can see all the current breakpoints. That way you can navigate away from them and you dont need to remember them to remove them. Double clicking on a breakpoint entry in the list will automatically load that breakpoint address. You can remove them in bulk by selecting them and pressing Delete. Or you can right click the selection and remove them that way. You can also rename each breakpoint entry for convenience.

### CODE REVIEW

File: rpcs3/rpcs3qt/debugger_frame.cpp [hunk 1]
```diff
@@ -473,9 +493,58 @@ void debugger_frame::EnableButtons(bool enable)
 
 void debugger_frame::ClearBreakpoints()
 {
+	//deleting the breakpoints here turns this function into an all purpose clear breakpoints instead of just clearing on emu stop
```
Comment (Marthgun) [inline rpcs3/rpcs3qt/debugger_frame.cpp]: minor quibble, you have 2 empty lines underneath this comment before the for loop.  most of the code base would have no lines, with comment directly above denoted loop.

Comment (MRG95) [reply rpcs3/rpcs3qt/debugger_frame.cpp]: Ah. I'll check more carefully next time :)

Comment (kd-11) [reply rpcs3/rpcs3qt/debugger_frame.cpp]: Remove the blank lines

Comment (MRG95) [reply rpcs3/rpcs3qt/debugger_frame.cpp]: Oh I didn't realize that I was the only one that could make the changes. I'm new to github. 1 sec

Comment (kd-11) [reply rpcs3/rpcs3qt/debugger_frame.cpp]: Technically I can edit the changeset for you but its better if the author does it themselves. That way, commit history is intact and git blame identifies the authors correctly.

**Pull request merged**